examples/tutorials/05_train_sklearn_models.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>I think it would be better to use one pipeline + pipeline config rather than two pipelines.</p><p>That would allow you to give different names for models, as well as simplify things overall.</p><p>Config must contain <code>model_name</code> and  <code>estimator</code> keys</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>